docker-compose-services/chromedriver/README.md

```diff
@@ -0,0 +1,25 @@
+# ChromeDriver
+
+This recipe adds a ChromeDriver container to a project.
+
+Among other things, this support Drupal's FunctionalJavascript tests.
+But it also supports Behat with WebDriver or any other use of headless Chrome.
+
+## Requirements
+
+None
+
+## Configuration
+
+None
+
+## Installation
+
+* Copy `docker-compose.chromedriver.yaml` to the `.ddev` folder of your project.
+* Set environment variables as appropriate. Examples are for Drupal Functional and FunctionalJavascript testing.
+  * To do this, copy (merge variables if one already exists) `docker-compose.environment.yaml` to the `.ddev` folder of your project.
+* Verify your phpunit.xml does not define environment variable for `MINK_DRIVER_ARGS_WEBDRIVER`. These should come from environment variable.
+* Start (or restart) DDEV to have the service initialized: `ddev start`
```

Comment:
Please add a simple test technique that people can use to verify whether it works or does not work, thanks.

Reply:
Done